archinstall/lib/installer.py

```diff
@@ -1017,6 +1126,7 @@ def add_bootloader(self, bootloader: Bootloader):
 		Archinstall supports one of three types:
 		* systemd-bootctl
 		* grub
+		* limine
```

Comment:
Maybe swap this to `* limine (beta)`, just for a bit?

Reply:
Sure!

Reply:
Fixed in https://github.com/archlinux/archinstall/pull/1815/commits/88123bcdf5dbfc965655fbc345c65b416799fe76.